QuizApp/App/Screens/Quiz/QuizListViewModel.swift

```diff
@@ -17,32 +18,43 @@ final class QuizListViewModel {
 
 extension QuizListViewModel {
 
+    @MainActor
```

Comment:
You know have two methods that are basically copy-paste. 
You can have just one private method that will have that code and two public methods that will call that private method with the appropriate arguments.

Reply:
I will fix that in the sections PR, so I don't have to solve merge conflicts later. :)